app/src/components/v-upload/v-upload.vue

```diff
@@ -121,6 +122,10 @@ export default defineComponent({
 			type: String,
 			default: undefined,
 		},
+		type: {
+			type: String as PropType<'image' | 'audio' | 'video'>,
+			default: undefined,
+		},
```

Comment:
We should use mime types here (eg `image/*`, `audio/*` etc)

Reply:
We should also make sure a list of options is allowed (eg `image/jpeg` _and_ `image/png`)